src/rgw/motr/gc/gc.cc

```diff
@@ -0,0 +1,58 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=2 sw=2 expandtab ft=cpp
+
+/*
+ * Garbage Collector implementation for the CORTX Motr backend
+ *
+ * Copyright (C) 2022 Seagate Technology LLC and/or its Affiliates
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation. See file COPYING.
+ *
+ */
+
+#include "gc.h"
```

Comment:
I think we should use file names as `motr_gc.h` and `motr_gc.cc` following the convention in RGW

Reply:
My thoughts:  since they have `rgw_gc.h/cc` files under `src/rgw` dir they needed this kind of naming 
since we are keeping it in separate folder `motr/gc`, I think `motr_gc.h/cc` will make it repetitive. `motr/gc/motr_gc.h`

probably here I can change the line to
```c++
#include "motr/gc/gc.h"
```
or can we ommit the gc folder? `motr/gc.h` or `motr/motr_gc.h`
what are your thoughts?

Reply:
We could have skipped the folder but since we already have one for `addb`, we can keep this structure as well.

Reply:
used `#include "motr/gc/gc.h"` for more readability.